src/delegate-proxy/templates/_helpers.tpl

```diff
@@ -63,4 +63,16 @@ Create the name of the service account to use
 
 {{- define "delegate-proxy.pullSecrets" -}}
 {{ include "common.images.pullSecrets" (dict "images" (list .Values.image) "global" .Values.global ) }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Create the name of the immutable delegate image to use
+*/}}
+{{- define "delegate-proxy.docker_image" -}}
+{{- $tag := printf "%s" .Values.image.tag }}
+{{- if .Values.global.useMinimalDelegateImage }}
+{{- $tag = printf "%s.minimal" $tag }}
```

Comment:
Just checked dockerhub, it is saved as _minimal.
Please synced with @faruk and discuss on what to do next.